decidim-core/app/cells/decidim/amendable/amend_button_card_cell.rb

```diff
@@ -10,29 +10,13 @@ def model_name
         model.model_name.human
       end
 
-      # REDESIGN_PENDING: deprecated
       def current_component
         model.component
       end
```

Comment:
If this method is deprecated should we remove it?

Reply:
I had to recover this method in https://github.com/decidim/decidim/pull/11685/commits/cda191610eb542843e7783a63bbc6ca619ec7b3f because at some point where this button is called the current component is required. The button makes via action_authorized_link_to, authorized_to and modal_path a `current_component.present?` call

Reply:
Ah OK, if its needed and the comment was wrong then lets leave it as is now